src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  margin-bottom: 24px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--1 .stars__star:nth-child(1),
+  &--2 .stars__star:nth-child(-n + 2),
+  &--3 .stars__star:nth-child(-n + 3),
+  &--4 .stars__star:nth-child(-n + 4),
+  &--5 .stars__star:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-left: 17px;
+  }
```

Comment:
This text shouldn't be related to the stars block, stars should only show them without any additional info